src/app_size_analyzer/cli.py

```diff
@@ -0,0 +1,273 @@
+"""Command-line interface for app size analyzer."""
+
+from __future__ import annotations
+
+import json
+import time
+from pathlib import Path
+from typing import Optional
+
+import click
+from rich.console import Console
+from rich.progress import Progress, SpinnerColumn, TextColumn
+from rich.table import Table
+
+from . import __version__
+from .analyzers.ios import IOSAnalyzer
+from .models.results import AnalysisResults
+from .utils.logging import setup_logging
+
+
+console = Console()
+
+
+@click.group(invoke_without_command=True)
+@click.option("--version", is_flag=True, help="Show version information and exit.")
+@click.pass_context
+def cli(ctx: click.Context, version: bool) -> None:
+    """App Size Analyzer - Analyze iOS and Android app bundle sizes."""
+    if version:
+        click.echo(f"App Size Analyzer v{__version__}")
+        ctx.exit()
+
+    if ctx.invoked_subcommand is None:
+        click.echo(ctx.get_help())
+
+
+@cli.command()
```

Comment:
This is a bit of a mouthful to read but this essentially sets up the `ios` subcommand, so these flags are only applicable to iOS.